conventions/src/main/kotlin/otel.java-conventions.gradle.kts

```diff
@@ -71,7 +71,7 @@ tasks.withType<JavaCompile>().configureEach {
           "-Xlint:-options",
 
           // Fail build on any warning
-          "-Werror"
+//          "-Werror"
```

Comment:
There's several "debug" changed in this PR you'll need to revert before it actually can receive approval; this is one of them.